packages/shared/lib/services/user.service.ts

```diff
@@ -66,10 +90,19 @@ class UserService {
         return result[0];
     }
 
-    async createUser(email: string, name: string, hashedPassword: string, salt: string, accountId: number): Promise<User | null> {
-        const result: Pick<User, 'id'> = await db.knex
-            .from<User>(`_nango_users`)
-            .insert({ email: email, name: name, hashed_password: hashedPassword, salt: salt, account_id: accountId }, ['id']);
+    async createUser(email: string, name: string, hashedPassword: string, salt: string, accountId: number, email_verified = true): Promise<User | null> {
+        const result: Pick<User, 'id'> = await db.knex.from<User>(`_nango_users`).insert(
```

Comment:
what about ```
const [userId] = await.db(....)
return userId ? this.getUserById(userId) : null
```

Reply:
Didn't do this but updated to be a bit clearer